clojure/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM clojure:latest
+
+RUN mkdir /deez
+WORKDIR /deez
+COPY . .
+# @fixme: this is somehow missing a few deps!
+RUN clj -A:deps -A:cljfmt -A:test -A:lint
+#RUN clj -X:deps prep
```

Comment:
I'm trying to cache the deps here, so that they don't need to be downloaded on every `docker run`.
However, this seems to miss a few? The commented-out command below also didn't work.